easybuild/framework/easyconfig/default.py

```diff
@@ -158,6 +158,11 @@
 
     # MODULES easyconfig parameters
     'whatis': [None, "List of brief (one line) package description entries", MODULES],
+    'usage': [None, "Usage instructions for the software", MODULES],
+    'docpaths': [None, "List of paths of installed documentation relative to package root dir", MODULES],
+    'docurls': [None, "List of URLs to package documentation", MODULES],
+    'support': [None, "Package-specific support/bug report address", MODULES],
+    'contact': [None, "Site contact for the package", MODULES],
```

Comment:
I think the `support` documentation needs to indicate that this is for upstream, not site-local support. Otherwise it's rather ambiguous.